packages/cli/src/cli.ts

```diff
@@ -75,7 +76,37 @@ class CLI {
                 },
             });
     }
-    static parseTestArgs(sywac : ISywac) : ISywac {
+    static parseRequireAppConfig(sywac: ISywac): ISywac {
+        return sywac
+            .array('-R, --require-config <path>', {
+                desc: 'Require app configuration file',
+                coerce: (strings: string[]) => {
```

Comment:
`return strings.map(path.resolve)` would do the job

Reply:
👍